pkg/interface/src/apps/chat/components/lib/chat-messages.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component, Fragment } from "react";
+import moment from "moment";
+
+import { Message } from "./message";
+
+type IMessage = Envelope & { pending?: boolean };
+
+
+export class ChatMessages extends Component {
```

Comment:
typing props would be nice. e.g. `extends Component<ChatMessagesProps>`

Reply:
Killed this whole file.